### ISSUE TITLE
Add support for multiple logins to user sessions

### ISSUE THREAD

**Issue by wgwz** [regen-network/regen-server#397]
We want to adjust the login system so that within a single user session cookie we have:
1. an active user
2. a list of active users

We want to have an endpoint that returns the current active user as an object from the session.
This endpoint should probably also return the list of active users.

We want to have an endpoint that switches the current active user in the session.
The active users list needs to be updated each time a particular login occurs.
This means for any supported login method; keplr, google, magic link etc. the list always has to be updated.
Additionally, the switch must check that user that is being switched is within the active users list.
Without such a check, there is a vulnerability.

For this task, we can pick a particular login type to support.
For now, it's probably simplest just to get this working with magic links.
Because the data model is going to be changing as it relates to keplr logins, we should wait for that.
This task doesn't have to be blocked if we just get things working with multiple magic logins.
And at that point it can be extended for the other login methods which are still under development and added for keplr once the data model is solid.

To summarize, this task can be thought of as handling these issues for magic links:
1. adjust the session cookie to have active user and active users list
2. create an API endpoint that allows setting the "active user" in session cookie
3. create an API endpoint that returns active user and active users list from the session cookie (via `req.session`)

See [this related task](https://github.com/regen-network/regen-server/issues/376) for much more detailed but not summarized discussion.
See [this experimental PR](https://github.com/regen-network/regen-server/pull/384) that does some of what the goals are for this task.

**Comment by wgwz** [comment]
@blushi i was between a 3 and 5 when i voted, so i would go with a 5 here
